show/main.py

```diff
@@ -991,6 +1001,53 @@ def portchannel(verbose):
     run_command(cmd, display_cmd=verbose)
 
 #
+# 'show interfaces loopback' group command
+#
+#
+@interfaces.command()
```

Comment:
I'm not sure if we want to have this command output. IMO, all of this is available in `show ip interfaces`. `show interfaces` is mostly the Layer 2 info. I think this command is redundant. What do you think?

Reply:
There was a internal request to add command to list loopback interfaces created since show ip interface would show only if IP was assigned. Hence we planned to add this command. If this is not necessary I can remove this command and update. Please let me know.

Reply:
IMO, it is better to avoid any duplicates. This file is already pretty big. Could you please remove this command since in most cases, loopback will be assigned with an IP address?

Reply:
Done